nall/platform.cpp

```diff
@@ -20,6 +20,14 @@ NALL_HEADER_INLINE auto setsockopt(int socket, int level, int option_name, const
   return ::setsockopt(socket, level, option_name, (const char*)option_value, option_len);
 }
 
+NALL_HEADER_INLINE auto usleep(unsigned int us) -> int {
```

Comment:
This is just what the MinGW-w64 implementation of usleep() happens to do; I didn't want to change behavior here. At some point, it would probably be wise to evaluate everywhere ares is doing sleeps and seek better alternatives.

Reply:
consider std::this_thread::sleep_for() ?

Reply:
Yeah, that seems like a good candidate, but I'd leave it for a separate change.